base/linalg/triangular.jl

```diff
@@ -1038,6 +1038,11 @@ for (f, g) in ((:/, :A_rdiv_B!), (:A_rdiv_Bc, :A_rdiv_Bc!), (:A_rdiv_Bt, :A_rdiv
         end
     end
 end
+### Fallbacks brought in from linalg/bidiag.jl while fixing #14506.
+# Eventually the above promotion methods should be generalized as
+# was done for bidiagonal matrices in #14506.
+At_ldiv_B(A::AbstractTriangular, B::AbstractVecOrMat) = At_ldiv_B!(A, copy(B))
+Ac_ldiv_B(A::AbstractTriangular, B::AbstractVecOrMat) = Ac_ldiv_B!(A, copy(B))
```

Comment:
Cheers, and agreed :). Hence the comment immediately preceding the two methods in question: I think the methods you linked require the `<:Number` restriction, at least for now. (In the interim I had forgotten the purpose of that comment --- thanks for reminding me!) But I held off with those modifications to the triangular code till the approach received approval in the bidiagonal case. Thanks again!